src/app/clusters/account-login-server/account-login-delegate.h

```diff
@@ -44,6 +44,8 @@ class Delegate
                                    const chip::CharSpan & tempAccountIdentifierString)                                 = 0;
     virtual void GetSetupPin(char * setupPin, size_t setupPinSize, const chip::CharSpan & tempAccountIdentifierString) = 0;
 
+    virtual uint16_t GetClusterRevision(chip::EndpointId endpoint) = 0;
```

Comment:
why is this a global getter for an account login delegate?

The description of the PR says "bumped up the revision to 2". However I am unclear why account-login now gets a dynamic revision number. Could you explain the changes and logic in the PR description so I understand the intent and usage of the changes?

Reply:
If we provide all attributes, why don't we provide feature map as well?
However I also wonder if the ember data model does not directly map global attributes such as revision or featuremap.

Reply:
The content apps that implement the account login delegate can have their own cluster revision. Account login delegate was not supporting this, so I had to extend the delegate so content app is able to return their version of account login implementation.